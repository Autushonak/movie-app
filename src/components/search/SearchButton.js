import styled from 'styled-components';
import { colors } from '../../constants/colors';

const SearchButton = styled.button`
  flex-grow: 0.5;
  background-color: ${colors.PINK};
  border: none;
  border-radius: 4px;
  text-transform: uppercase;
  color: ${colors.WHITE};
  padding: 15px 60px;
  text-align: center;
  font-size: 16px;
`;

export default SearchButton;
