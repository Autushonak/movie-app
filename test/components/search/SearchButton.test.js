import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import SearchButton from '../../../src/components/search/SearchButton';

describe('SearchButton ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<SearchButton>Search</SearchButton>);
    expect(component).toMatchSnapshot();
  });
});
