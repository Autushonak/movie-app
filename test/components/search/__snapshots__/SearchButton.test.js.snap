// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton  should be rendered correctly 1`] = `
.c0 {
  -webkit-box-flex: 0.5;
  -webkit-flex-grow: 0.5;
  -ms-flex-positive: 0.5;
  flex-grow: 0.5;
  background-color: #F65261;
  border: none;
  border-radius: 4px;
  text-transform: uppercase;
  color: #FFFFFF;
  padding: 15px 60px;
  text-align: center;
  font-size: 16px;
}

<button
  className="c0"
>
  Search
</button>
`;
